betting-house/local-deployment/docker-compose.yml

```diff
@@ -2,11 +2,14 @@ version: '2.2'
 services:
   postgres-betting-db:
     image: postgres:latest
+    volumes:
+      - ../common-deployment:/init-scripts
     ports:
       - 5432:5432
     environment:
       POSTGRES_USER: betting
       POSTGRES_PASSWORD: betting
+    entrypoint: /bin/bash -c "apt-get update && apt-get install -y gosu && /init-scripts/init_psql.sh"
```

Comment:
```suggestion
    entrypoint: /init-scripts/init_psql.sh
```

Reply:
Thank you very much for this improvement, Vasile. That's very kind of you.
I left a couple of suggestions you could follow. Just nitpicking. I hope you don't mind.